sectionsUtilityTool/src/main/webapp/section_data/courses-amanjo.json

```diff
@@ -0,0 +1 @@
+[{"account_id":281,"course_code":"THTREMUS 233","default_view":"syllabus","enrollment_term_id":39,"id":466,"is_public":null,"name":"Actg\u0026Black Exp","start_at":"2015-01-05T01:20:11Z","end_at":null,"public_syllabus":false,"storage_quota_mb":1000,"apply_assignment_group_weights":false,"calendar":{"ics":"https://umich.instructure.com/feeds/calendars/course_b4ENPXPOOnSHLNeDa1apKXmRs0iTAZh2mzg8hr0s.ics"},"sis_course_id":"20200116962","integration_id":null,"enrollments":[{"type":"teacher","role":"TeacherEnrollment","role_id":15,"enrollment_state":"active"},{"type":"teacher","role":"TeacherEnrollment","role_id":15,"enrollment_state":"active"}],"hide_final_grades":false,"workflow_state":"available"},{"account_id":281,"course_code":"THTREMUS 340","default_view":"syllabus","enrollment_term_id":39,"id":610,"is_public":null,"name":"Devising Theatre","start_at":"2015-01-07T20:18:23Z","end_at":null,"public_syllabus":false,"storage_quota_mb":1000,"apply_assignment_group_weights":false,"calendar":{"ics":"https://umich.instructure.com/feeds/calendars/course_6p4cpJLVjRGYIbkqSaKWavcP5XefumBhZcjCx4iz.ics"},"sis_course_id":"20200388391","integration_id":null,"enrollments":[{"type":"teacher","role":"TeacherEnrollment","role_id":15,"enrollment_state":"active"},{"type":"teacher","role":"TeacherEnrollment","role_id":15,"enrollment_state":"active"}],"hide_final_grades":false,"workflow_state":"available"}]
```

Comment:
Yesterday David told me that these JSON files are actually for testing, not production use.  They should be moved into a directory called "tests" to make that clear.